app/models/induction_programme.rb

```diff
@@ -22,7 +22,8 @@ class InductionProgramme < ApplicationRecord
   has_many :transferred_induction_records, -> { transferred }, class_name: "InductionRecord"
   has_many :participant_profiles, through: :active_induction_records
   has_many :current_participant_profiles, through: :current_induction_records, source: :participant_profile
-
+  has_one :lead_provider, through: :partnership
```

Comment:
do we still need https://github.com/DFE-Digital/early-careers-framework/pull/2225/files#diff-3e4c2359566ad9ebf06892b48edf98cbf3cb2296b4d3b37b292e2a7527e4e52fL27 now we have the `has_one` through set?